models/case-audit.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (sequelize, DataTypes) => {
+  const CaseAudit = sequelize.define('CaseAudit', {
+    twilioWorkerId: DataTypes.STRING,
+    previousValue: DataTypes.JSONB,
+    newValue: DataTypes.JSONB,
+    /**
+     * In theory, we shouldn't need to explicitly declare 'caseId' here,
+     * since the association CaseAudit.belongsTo(Case) would handle it.
+     * But sequelize was NOT inserting 'caseId' when CREATING a CaseAudit.
+     */
+    caseId: DataTypes.INTEGER,
+  });
```

Comment:
Would we want to capture a revision id?  Would we want to have the timestamp of the change at the top level here, or is having it in `newValue.dataValues.updatedAt` sufficient?

Reply:
`CaseAudit` will have by default `createdAt` and `updatedAt` columns as the other tables.

Reply:
Okay.  Seems like that should work fine.